python/tvm/tir/ir_builder.py

```diff
@@ -249,17 +249,17 @@ def for_range(self, begin, end, name="i", dtype="int32", for_type="serial"):
         extent = end if begin == 0 else (end - begin)
 
         def _exit_cb():
-            if for_type == "serial":
-                for_type_id = 0
-            elif for_type == "parallel":
-                for_type_id = 1
-            elif for_type == "vectorize":
-                for_type_id = 2
-            elif for_type == "unroll":
-                for_type_id = 3
+            if kind == "serial":
+                kind_id = 0
+            elif kind == "parallel":
+                kind_id = 1
+            elif kind == "vectorize":
+                kind_id = 2
+            elif kind == "unroll":
+                kind_id = 3
```

Comment:
```suggestion
            if kind == "serial":
                kind_id = ForKind.SERIAL
            elif kind == "parallel":
                kind_id = ForKind.PARALLEL
            elif kind == "vectorize":
                kind_id = ForKind.VECTORIZED
            elif kind == "unroll":
                kind_id = ForKind.UNROLLED
```

Will it be better to also use ForKind as the input of this function?

Reply:
We are trying to be consistent with the old API for now. So let us keep it as it is.